z3-sys/build.rs

```diff
@@ -31,16 +60,26 @@ fn main() {
         "symbol_kind",
     ] {
         let mut enum_bindings = bindgen::Builder::default()
-            .header(&header)
+            .header(header)
             .parse_callbacks(Box::new(bindgen::CargoCallbacks))
             .generate_comments(false)
             .rustified_enum(format!("Z3_{}", x))
             .allowlist_type(format!("Z3_{}", x));
-        if env::var("TARGET").unwrap() == "wasm32-unknown-emscripten" {
-            enum_bindings = enum_bindings.clang_arg(format!(
-                "--sysroot={}/upstream/emscripten/cache/sysroot",
-                env::var("EMSDK").expect("$EMSDK env var missing. Is emscripten installed?")
-            ));
+        let target = env::var("TARGET").unwrap();
```

Comment:
I'd rather have the emscripten / webassembly changes as a separate PR. I'm less sure about what's happening here and it isn't directly related to the `vcpkg` work (to me ... my main interest is in using `vcpkg` to improve Windows build support where we don't have an option to use a system-installed Z3).

Reply:
Reverted the commit. I'll have more tests and then create a new PR.